README.md

```diff
@@ -41,20 +41,20 @@ Below are the various ways of generating images:
   [info] Loading settings for project root from dependencies.sbt,build.sbt ...
   [info] Set current project to play-docker-seeder (in build file:/Users/ivan/Code/env/inventure/apps/play-docker-seeder/)
   [info] ### Inquiring versions
-  Base Docker Image [debian:bullseye-20240612-slim] : debian:bullseye-20240612-slim
+  Base Docker Image [debian:bullseye-20240701-slim] : debian:bullseye-20240612-slim
   Play! version [2.9.4] : 2.9.4
   Scala version [2.13.14] : 2.13.14
   Java version [11.0.23-amzn] : 11.0.23-amzn
   Play-Slick version [5.3.0] :
-  Sbt version [1.10.0] :
+  Sbt version [1.10.1] :
   Docker registry [ivanoronee] :
   [info] Working with versions:
-  [info] - base-image => debian:bullseye-20240110-slim
+  [info] - base-image => debian:bullseye-20240701-slim
   [info] - play       => 2.9.4
   [info] - scala      => 2.13.14
   [info] - java       => 11.0.23-amzn
   [info] - play-slick => 5.3.0
```

Comment:
Let's update this as well https://github.com/playframework/play-slick/releases/tag/5.3.1